airflow/models/baseoperator.py

```diff
@@ -651,6 +653,10 @@ class derived from this one results in the creation of a task object,
         that is visible in Task Instance details View in the Webserver
     :param doc_yaml: Add documentation (in YAML format) or notes to your Task objects
         that is visible in Task Instance details View in the Webserver
+    :param logger_name: Name of the logger used by the Operator to emit logs.
+        Default is "airflow.task.operators". If set to `None`, the logger name will fall back
+        to `{class.__module__}.{class.__name__}` (e.g. SimpleHttpOperator will have
```

Comment:
Similar NIT -> this docstring is now not entirely accurate (lack of "airflow.task.operarors" prefix.